popup.css

```diff
@@ -0,0 +1,498 @@
+  color: #fff;
+}
+
+#projects {
+  width: 100%;
+  margin: 0%;
+  display: grid;
+  grid-template-rows: repeat(3, auto);
+  align-items: space-between;
+  padding-top: 10%;
+}
+
+.data {
+  margin-top: 5%;
+  margin-bottom: 8%;
+  padding: 5%;
+  display: flex;
+  flex-direction: column;
+  align-items: left;
+  justify-content: space-between;
+}
+
+.canopy {
+  font-style: normal;
+  font-weight: 500;
+  font-size: 18px;
+  line-height: 16px;
+  margin-top: 1%;
+  color: #344563;
+}
+
+.backend {
+  font-style: normal;
+  font-weight: 400;
+  font-size: 18px;
+  line-height: 16px;
+  margin-top: 1%;
+  color: #7a868a;
+}
+
+.year {
+  font-style: normal;
+  font-weight: 400;
+  font-size: 18px;
+  line-height: 16px;
+  margin-top: 1%;
+  color: #7a868a;
+}
+
+.tools {
+  list-style: none;
+  padding-left: 0%;
+  width: 100%;
+  margin-top: 1%;
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+}
+
+.tools li {
+  display: inline-block;
+  justify-content: center;
+  padding: 4px 12px;
+  margin-right: 2.5px;
+  border-radius: 8px;
+  background-color: #ebebff;
+  font-style: normal;
+  font-weight: 500;
+  font-size: 12px;
+  line-height: 16px;
+  color: #6070ff;
+  letter-spacing: 0.03em;
+}
+
+.see-project {
+  margin-top: 5px;
+  width: 44%;
+  background-color: white;
+  border: 1px solid #6070ff;
+  border-radius: 6px;
+  padding: 16px;
+  gap: 12px;
+  align-items: center;
+  font-style: normal;
+  font-weight: 500;
+  font-size: 17px;
+  line-height: 24px;
+  letter-spacing: 0.03em;
+  color: #396df2;
+}
+
+.skills-block {
+  display: flex;
+  flex-direction: row;
+  justify-content: space-between;
+}
+
+.space {
+  height: 1px;
+  width: 95%;
+  margin-top: 7%;
+  margin-left: -4%;
+  background-color: #dfe1e5;
+}
+
+.pointer {
+  position: absolute;
+  right: 0;
+  padding-right: 25%;
+  padding-top: 2%;
+}
+
+@media screen and (min-width: 768px) {
+  .image-div {
+    display: none;
+  }
+
+  .see-project:hover {
+    background: #396df2;
+    color: #fff;
+    transition: 0.8s;
+  }
+
+  .project-content-1 {
+    width: 50%;
+    display: flex;
+    flex-direction: column;
+    justify-content: space-between;
+    margin-top: 0;
+  }
+
+  .project-content-2 {
+    width: 50%;
+    display: flex;
+    flex-direction: column;
+    justify-content: space-between;
+    margin-top: 0;
+  }
+
+  .project-content-4 {
+    width: 50%;
+    display: flex;
+    flex-direction: column;
+    justify-content: space-between;
+    margin-top: 0;
+  }
+
+  .iconicon {
+    display: none;
+  }
+
+  nav a {
+    text-decoration: none;
+  }
+
+  .header-details > a {
+    margin-right: 20px;
+  }
+
+  .mobile-cat {
+    font-size: 15px;
+  }
+
+  .modal-button {
+    font-size: 12px;
+    border-radius: 8px;
+    width: 45%;
+    transition: background-color 0.3s ease;
+  }
+
+  .modal-button:hover {
+    background-color: #e6e6e6;
+    text-decoration: underline;
+  }
+
+  .modal-body {
+    width: 1156px;
+    height: 800px;
+  }
```

Comment:
- [ ] To make the `pop-up` menu responsive across all `screen` sizes kindly consider using the percentage`(%)` unit for your `width` instead of pixel`(px)`. This approach ensures that the menu adapts and scales appropriately based on the available screen width, offering a better user experience across different devices and resolutions. By using percentage-based widths, the pop-up menu will be more responsive and fluid, adjusting its size proportionally to fit various screen sizes.
`HINT`: Set `width: 80%`;

- [ ]  Also, Kindly consider adjusting the `height` of the pop-up so that the user can see the whole content on it. 


<table>
<tr>
<th align=“middle”>Figma design</th>
<th align=“middle”>Your design</th>
</tr>
<tr>
<td>
<img width="535" alt="Screenshot 2023-06-29 at 11 43 52" src="https://github.com/Edgahkipkemoi/mobile/assets/107192076/fea5152d-a85f-49ca-9a6d-7838a84437d2">
</td>
<td>
<img width="1229" alt="Screenshot 2023-06-29 at 11 42 51" src="https://github.com/Edgahkipkemoi/mobile/assets/107192076/8e8fbe35-090d-4056-b2c5-11649573daef">
</td>
</tr>
</table>